R/IDateTime.R

```diff
@@ -240,6 +240,22 @@ rep.ITime = function (x, ...)
   class(y) = "ITime"   # unlass and rep could feasibly not copy, hence use class<- not setattr()
   y
 }
+                           
+round.ITime <- function(x, digits = c("hours", "minutes")) 
+{
+  (setattr(switch(match.arg(digits),
```

Comment:
needs to be added in `NAMESPACE`, needs tests, and please add something to NEWS; please see:

https://github.com/Rdatatable/data.table/wiki/Contributing